csharp/ql/test/resources/stubs/Microsoft.Extensions.Configuration/8.0.0/Microsoft.Extensions.Configuration.cs

```diff
@@ -0,0 +1,102 @@
+namespace Microsoft.Extensions.Configuration
```

Comment:
The subfolders of `stubs` primary contains auto generated stubs and the files in the stubs folder are "handwritten".
In `csharp/scripts/stubs` there are scripts for generating stubs from NuGet packages.
However, in this case I think there already exists stubs for these methods. They are in the ASP.NET framework instead.